kandinsky2/kandinsky2_1_model.py

```diff
@@ -30,6 +30,8 @@ def __init__(
     ):
         self.config = config
         self.device = device
+        if not torch.has_cuda:
```

Comment:
However, this does not work upon attempting to run on CPU when having CUDA-enabled version of PyTorch installed (an example use-case: _I have a CUDA device, but I want to generate a higher-resolution image, and I don't have enough VRAM to do that on the GPU_). Maybe we should check `device` directly (at least this workaround works for me)?
```suggestion
        if device == "cpu":
```

Reply:
Good point, but maybe `device!="cuda" then, because same is needed for "mps" device (Apple Silicon), I don't know about others.